### DISCUSSION TITLE
Lesson 9: Unable to interact with my deployed contract on sepolia

### DISCUSSION THREAD

**Discussion by DownShiftDom** [Cyfrin/foundry-full-course-cu#356] [Q&A] [answered]
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/26583886/9113adbc-19f5-4652-9007-9d67b0def13a)
Here's my contract: https://sepolia.etherscan.io/address/0x3902bdb6c813e963490c72a9fd6def2a97bdca66#code

Here's a screenshot of when I try to interact with enterRaffle: 
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/26583886/f67c79ff-92d9-454c-904b-e5e1b56acb48)

**Comment by shikhar229169** [comment, marked as answer]
As I can see in your contract the minimum entrance fees is 0.1 ETH (10 to the power 17 wei)
